accounts/models.py

```diff
@@ -3,19 +3,37 @@
 from django.contrib.auth import get_user_model
 from django.contrib.auth.models import AbstractUser
 from django.db import models
+from phonenumber_field.modelfields import PhoneNumberField
 
 
 class User(AbstractUser):
     # implicit username, email, first_name, and last_name fields
     # from AbstractUser that contains the user's PennKey
     pennid = models.IntegerField(primary_key=True)
     uuid = models.UUIDField(unique=True, default=uuid.uuid4, editable=False)
+    preferred_name = models.CharField(max_length=100, blank=True)
 
 
 class Student(models.Model):
     user = models.OneToOneField(get_user_model(), on_delete=models.DO_NOTHING)
     major = models.CharField(max_length=255, blank=True)
     school = models.CharField(max_length=255, blank=True)
+    graduation_year = models.CharField(max_length=4, blank=True)
 
     def __str__(self):
         return self.user.username
+
+
+class Email(models.Model):
+    user = models.ForeignKey(get_user_model(), related_name="emails", on_delete=models.DO_NOTHING)
```

Comment:
I think on_delete for this should probably be cascade. User accounts don't really get deleted, but I think it makes sense to have the email object deleted if the user object is.